vault/resource_aws_secret_backend.go

```diff
@@ -320,23 +300,10 @@ func awsSecretBackendRead(ctx context.Context, d *schema.ResourceData, meta inte
 	if err := d.Set(consts.FieldPath, path); err != nil {
 		return diag.FromErr(err)
 	}
-	if err := d.Set(consts.FieldDescription, mount.Description); err != nil {
-		return diag.FromErr(err)
-	}
-	if err := d.Set(consts.FieldDefaultLeaseTTL, mount.Config.DefaultLeaseTTL); err != nil {
-		return diag.FromErr(err)
-	}
-	if err := d.Set(consts.FieldMaxLeaseTTL, mount.Config.MaxLeaseTTL); err != nil {
-		return diag.FromErr(err)
-	}
-	if err := d.Set(consts.FieldLocal, mount.Local); err != nil {
+
+	if err := readMount(d, meta, true); err != nil {
```

Comment:
Should we also pass a context here to `readMount()`?

Reply:
Updated all read instances to also use the context